packages/@aws-cdk/cx-api/lib/versioning.ts

```diff
@@ -64,6 +64,11 @@ export function upgradeAssemblyManifest(manifest: AssemblyManifest): AssemblyMan
     manifest = justUpgradeVersion(manifest, '1.10.0');
   }
 
+  if (manifest.version === '1.10.0') {
```

Comment:
Thank you for getting where I was going with this! 

(Now that we've done it twice it will be a tradition ;) )